### PR TITLE
[GCI 2017] fix the upload menu

### LINKED ISSUE #17: The Command -> Upload menu does nothing
While trying to upload files to test my layout management code, I found out that the Command -> Upload menu doesn't do anything.

### PR DESCRIPTION
**Pull request by sfanxiang**
Fixes #17.

### CODE REVIEW

**Comment by janus2** [comment]
Could you rename the button "open" to "upload" with the method SetButtonLabel?

**Comment by sfanxiang** [comment]
@janus2 Actually I think "upload" makes it clear what will happen after the user chooses the file, and is more analogical to "download".

File: src/FTPWindow.cpp [hunk 1]
```diff
@@ -104,7 +104,10 @@ TFTPWindow::TFTPWindow(BRect frame, const char *name)
 	float minW, minH, maxW, maxH;
 	this->GetSizeLimits(&minW, &maxW, &minH, &maxH);
 	this->SetSizeLimits(280, maxW, 250, maxH);
-	
+
+	fOpenPanel.SetTarget(BMessenger(this));
+	fOpenPanel.SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE("Upload"));
+
```
Comment (humdingerb) [inline src/FTPWindow.cpp]: We should change the panel's window title as well:
fOpenPanel.Window()->SetTitle(B_TRANSLATE("FtpPositive: Upload"));

Comment (janus2) [reply src/FTPWindow.cpp]: Good point...

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**